cspell/.cspell.json

```
@@ -896,6 +905,7 @@
         "vindices",
         "vios",
         "visualstudioexptteam",
+        "vline",
```

```suggestion
        "vlines",
```
`vlines`? <https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.vlines.html#matplotlib-pyplot-vlines>